app/Distance.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace ConverterApp;
+
+class Distance extends Value
+{
+  public function __construct($value,$unit)
+  {
+     if($value<0)
+      {
+     	 throw new \InvalidArgumentException("Please enter valid argument.");
```

Comment:
Message should be as clear as possible. In this case you can say `Value can't be negative`